client/my-sites/upgrades/domain-management/dns/dns-add-new.jsx

```diff
@@ -166,7 +169,7 @@ const DnsAddNew = React.createClass( {
 
 				<FormFooter>
 					<FormButton
-						disabled={ formState.isSubmitButtonDisabled( this.state.fields ) || this.props.isSubmittingForm }
+						disabled={ this.state.show ? isSubmitDisabled : false }
```

Comment:
Ah yes, remembered that was the case. You can leave it as is right now. But I feel like we need to address this at some point.